prow/cmd/deck/static/command-help.html

```diff
@@ -5,7 +5,7 @@
         <link id="favicon" rel="icon" type="image/png" href="favicon.ico">
         <link rel="stylesheet" type="text/css" href="style.css">
         <link rel="stylesheet" type="text/css" href="extensions/style.css">
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,700">
```

Comment:
this is wrong, we need to load the font weights for bold and normal.

Reply:
same below, I added this intentionally :-)

Reply:
ack